### ISSUE TITLE
TODO : Maintenance Site

### ISSUE THREAD

**Issue by anasrar** [anasrar/Samehadaku-Lewatin#4]
# TODO List :
⏳ make command list all anime episode by name (ongoing, but i'm to lazy to make it)
# Safelink & Shortlink :
List already done
- ✅ coeg.in
- ✅ telondasmu.com
- ✅ tetew.info
- ✅ greget.space
- ✅ siherp.com
- ✅ njiir.com
- ✅ eue.siherp.com
- ✅ anjay.info
- ✅ ahexa.com
- ✅ bucin.net

**Comment by anasrar** [comment]
### Update
- Search Command Deprecated
- Fix From Command

**Comment by anasrar** [comment]
**TODO** :
✔ break new safelink system success

**Comment by anasrar** [comment]
**ADD**
new short url : ahexa.com

**Comment by anasrar** [comment]
### Update
- Change Site URL to samehada.tv

**Comment by anasrar** [comment]
ADD
new short url : bucin.net

**Comment by anasrar** [comment]
waiting change url due blocked by internet positif

**Comment by anasrar** [comment]
# INFO
if in one week the site not change url or domain, this repo will be archive

thanks for people use and give me feedback for along time
